spec/support/simplecov_setup.rb

```diff
@@ -15,3 +15,9 @@
     add_filter "/spec/"
   end
 end
+
+# https://codecov.io/guide
+if ENV['CI']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.